[core]/es_extended/client/functions.lua

```diff
@@ -406,6 +406,12 @@ function ESX.Game.SpawnVehicle(vehicle, coords, heading, cb, networked)
     local model = type(vehicle) == 'number' and vehicle or joaat(vehicle)
     local vector = type(coords) == "vector3" and coords or vec(coords.x, coords.y, coords.z)
     networked = networked or true
+    local player_coords = GetEntityCoords(ESX.PlayerData.ped)
```

Comment:
vector or player_coords or ESX.PlayerData.ped is nil break code

Reply:
?

Reply:
You calculate distance but if player_coords or vector is nil break code. You should handle nil values.

Reply:
use camelCase please.